SpatialConvolutionLocal.lua

```diff
@@ -0,0 +1,166 @@
+local SpatialConvolutionLocal, parent = torch.class('nn.SpatialConvolutionLocal', 'nn.Module')
+
+function SpatialConvolutionLocal:__init(nInputPlane, nOutputPlane, iW, iH ,kW, kH, dW, dH, padW, padH)
+   parent.__init(self)
+
+   dW = dW or 1
+   dH = dH or 1
+
+   self.nInputPlane = nInputPlane
+   self.nOutputPlane = nOutputPlane
+   self.kW = kW
+   self.kH = kH
+   self.iW = iW
+   self.iH = iH
+
+   self.dW = dW
+   self.dH = dH
+   self.padW = padW or 0
+   self.padH = padH or self.padW
+   self.oW = math.ceil((self.padW * 2 + iW - self.kW) / self.dW + 1)
```

Comment:
in both `SpatialConvolution` and `SpatialConvolutionMM` the output dimensions are [computed using `floor`](https://github.com/torch/nn/blob/master/generic/SpatialConvolutionMM.c#L207-L208). Wouldn't it be better to keep it consistent here as well ?